build.gradle

```diff
@@ -18,6 +18,12 @@ task bundleParticipantUI(type: NpmTask, dependsOn: [npmInstall, buildUICore]) {
     args = ['--workspace=ui-participant', 'run', 'build']
 }
 
+task printVersion {
+    doLast {
+        println gradle.ext.releaseVersion
+    }
+}
+
```

Comment:
oddly, this gradle task wasn't returning anything when running in CI. so i switched to setting the `ci` tag when we do the initial docker build